CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+/responsibleai @gaugup @gregorybchris @imatiach-msft @riedgar-ms @romanlutz
```

Comment:
can you add me here as well?

Reply:
can you add /erroranalysis too?

Reply:
I think we can make the format a bit better.  Check out the code owners page in mmlspark:

https://github.com/Azure/mmlspark/blob/master/CODEOWNERS

It has really nice formatting with some comments